example/issue_comment.go

```diff
@@ -65,16 +65,21 @@ func (h *PRCommentHandler) Handle(ctx context.Context, eventType, deliveryID str
 
 	repoOwner := repo.GetOwner().GetLogin()
 	repoName := repo.GetName()
-	author := event.GetComment().GetUser()
+	author := event.GetComment().GetUser().GetLogin()
 	body := event.GetComment().GetBody()
 
+	if author[len(author)-5:] == "[bot]" {
+		zerolog.Ctx(ctx).Debug().Msg("Issue comment was created by a bot")
```

Comment:
The `logger` should already have all the context necessary. I think you just want:
```
logger.Debug().Msg("Issue comment was created by a bot")
```